app/views/steps/postcode_screen/error_but_continue/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title t('.page_title') %>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <%= step_header %>
+
+    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
+
+    <p class="lede"><%=t '.lead_text' %></p>
+
+    <!-- TODO: Add the copy needed, and adapt the button linking to another step -->
+
+    <div class="xform-group form-submit">
+      <%= link_to t('.go_back'), edit_steps_postcode_screen_children_postcodes_path, class: 'link-back link-button' %>
```

Comment:
Wouldn't be better to just use the 'back' link we have in the header in all the steps? `<%= step_header %>`

Reply:
sure -  I went for the second explicit link as the copy is implying a choice between two possible actions, so presenting the options next to each other seemed to make sense. I've taken it out now though.